doc/classes/CompositorEffect.xml

```diff
@@ -57,6 +57,17 @@
 			var render_scene_buffers : RenderSceneBuffersRD = render_data.get_render_scene_buffers()
 			var roughness_buffer = render_scene_buffers.get_texture("forward_clustered", "normal_roughness")
 			[/codeblock]
+			The raw normal and roughness buffer is stored in an optimized format, different than the one available in Spatial shaders. When sampling the buffer from a compute shader, a conversion function must be applied. Use this function, copied from [url=https://github.com/godotengine/godot/blob/master/servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered_inc.glsl]here[/url]:
```

Comment:
```suggestion
			The raw normal and roughness buffer is stored in an optimized format, different than the one available in Spatial shaders. When sampling the buffer, a conversion function must be applied. Use this function, copied from [url=https://github.com/godotengine/godot/blob/da5f39889f155658cef7f7ec3cc1abb94e17d815/servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered_inc.glsl#L334-L341]here[/url]:
```

Reply:
Two changes:
1. this is needed no matter how you sample the texture (reading from fragment shader, copying to CPU then reading, or compute shader)
2. We can use a permalink to the exact code. We shouldn't use links to the master branch because the master branch is always changing

Reply:
In the future if I need to link to the godot source, should I permalink to the current master blob at the time of writing?